src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write code here
+const listOfEmployees = document.querySelector('ul');
+const arrOfEmployees = [...document.querySelectorAll('li')];
+
+function convertToNumber(value) {
+  return +value.slice(1).split(',').join('');
+}
+
+function sortList(item) {
+  const sortItem = item.sort((a, b) =>
+    convertToNumber(b.dataset.salary) - convertToNumber(a.dataset.salary)
+  );
+
+  listOfEmployees.append(...sortItem);
+};
```

Comment:
`sortList `function should ONLY sort the list! It is important. Everyone should understand what the function does from its name